### PR TITLE
fix for empty plugins property in serverless.yml

### PR DESCRIPTION
**Pull request by ac360**
## What did you implement:

Trivial fix to the bug in #2628

### CODE REVIEW

File: lib/classes/PluginManager.js [hunk 1]
```diff
@@ -73,7 +73,7 @@ class PluginManager {
   }
 
   loadServicePlugins(servicePlugs) {
-    const servicePlugins = (typeof servicePlugs !== 'undefined' ? servicePlugs : []);
+    const servicePlugins = (servicePlugs && typeof servicePlugs !== 'undefined' ? servicePlugs : []);
```
Comment (dougmoscrop) [inline lib/classes/PluginManager.js]: The `!== undefined` part is never going to be evaluated

Comment (dougmoscrop) [inline lib/classes/PluginManager.js]: what about?
```javascript

if (servicePlugs && !Array.isArray(servicePlugs) {
  throw ...
}
const servicePlugins = servicePlugs || []
```

Comment (ac360) [inline lib/classes/PluginManager.js]: Yes, I changed it before seeing this.  Good idea checking if it's an array, but I don't think it should throw an error.  It's fine if the `plugins` property is empty.

**Comment by flomotlik** [comment]
validated, thanks for the change

**Pull request merged**